workflow/rules/common.smk

```diff
@@ -9,6 +9,7 @@ validate(config, schema="../schemas/config.schema.yaml")
 
 wildcard_constraints:
     sample="[a-zA-Z_0-9-]+",
+    group="[a-zA-Z_0-9-]+",
```

Comment:
Maybe constrain this to the samples and groups read from the `samples.tsv` file?

Reply:
Good point.

Reply:
Done.